generated/.tailcallrc.schema.json

```diff
@@ -1192,6 +1192,38 @@
       },
       "type": "object"
     },
+    "Link": {
```

Comment:
Need to add `Link` to the whitelist of type that we want to add in tailcallrc.graphql. The code is in autogen.